cirq/ops/common_channels.py

```diff
@@ -89,10 +89,13 @@ def __str__(self) -> str:
                 .format(self._p_x, self._p_y, self._p_z))
 
     def _circuit_diagram_info_(self,
-        args: protocols.CircuitDiagramInfoArgs) -> protocols.CircuitDiagramInfo:
-        return protocols.CircuitDiagramInfo(
-            wire_symbols=('A',),
-            exponent='({!r},{!r},{!r})'.format(self._p_x, self._p_y, self._p_z))
+        args: protocols.CircuitDiagramInfoArgs) -> str:
+        return 'A({!r},{!r},{!r})'.format(self._p_x, self._p_y, self._p_z)
+
+
+def asymmetric_depolarize(p_x, p_y, p_z):
```

Comment:
Is this actually sufficient to specify any kind of depolarization? For example, measurement along the X+Z axis doesn't seem to be representable without introducing surrounding unitaries.